q2_assembly/quast/quast.py

```diff
@@ -382,4 +412,34 @@ def evaluate_contigs(
         # 3. read it as a pandas dataframe then we create the QUASTResults
         tabular_results = ctx.make_artifact("QUASTResults", report_df)
 
-    return tabular_results, visualization
+        # 4. create the Genomes
+        if not references:
+            try:
+                genomes = ctx.make_artifact("GenomeData[DNASequence]", genomes_dir)
+            except ValidationError as e:  # corrupt files
+                if "Missing one or more" in str(e):
+                    warn(
+                        "WARNING: QUAST did not download any genomes. The returned "
+                        "GenomeData[DNASequence] artifact is empty. Please check "
+                        "the network connection or provide the reference genomes."
+                    )
+                else:
+                    warn(
+                        "WARNING: There is a problem with the downloaded genome files. "
+                        "The returned GenomeData[DNASequence] artifact will be empty. "
+                        "You can either try again or provide the reference genomes."
+                    )
```

Comment:
I think this last line is a bit confusing. You should rather say something like:
```suggestion
                    warn(
                        "There was a problem with the genome files downloaded by QUAST. "
                        "The returned GenomeData[DNASequence] artifact will be empty. "
                        f"The original error was: '{e}'."
                    )
```